### PR TITLE
Fix indentation in some description

### PR DESCRIPTION
**Pull request by amenophis**
**- What I did**
Fix the indentation to allow jane-openapi generate to work

### CODE REVIEW

**Review by thaJeztah — APPROVED** [review]
thanks! changes LGTM, but I left some suggestions (let me know if those work for you; I'm also ok with merging as-is).

Curious why jane-openapi failed on this (as I _think_ the previous version was valid YAML)

File: api/swagger.yaml [hunk 1]
```diff
@@ -6386,9 +6386,9 @@ paths:
         - name: "networkmode"
           in: "query"
           description: "Sets the networking mode for the run commands during
```
[the remark this answers is not in the document]
Comment (fntlnz) [reply api/swagger.yaml]: Isn't the fallthrough case also allowing the use of network IDs? I would specify that

[the remark this answers is not in the document]
Comment (thaJeztah) [reply api/swagger.yaml]: Yes, I was already looking at this part; also because this list is likely not complete (taking Windows into account)

**Comment by AkihiroSuda** [comment]
ping @amenophis

**Comment by thaJeztah** [comment]
carried in https://github.com/moby/moby/pull/39519

**Pull request closed without merging**